src/main/java/edu/tamu/scholars/middleware/view/model/ExportFieldView.java

```diff
@@ -17,7 +17,7 @@ public class ExportFieldView extends FieldView {
 
     public ExportFieldView() {
         super();
-        limit = 5;
+        limit = Integer.MAX_VALUE;
```

Comment:
This only effect exports. Exports will be exhaustively tested.

- individual person display has two docx
- directory and discovery views have csv

and now an export for API only